dali/sasha/saxref.cpp

```diff
@@ -2363,14 +2363,13 @@ class CSashaExpiryServer: public ISashaServer, public Thread
             if (stopped)
                 break;
             const char *lfn = expirylist.item(i);
-            PROGLOG(LOGPFX2 "Deleting %s",lfn);
             try
             {
                 /* NB: 0 timeout, meaning fail and skip, if there is any locking contention.
                  * If the file is locked, it implies it is being accessed.
                  */
                 queryDistributedFileDirectory().removeEntry(lfn, udesc, NULL, 0, true);
-                PROGLOG(LOGPFX2 "Deleted %s",lfn);
+                OWARNLOG(LOGPFX2 "Deleted %s",lfn);
```

Comment:
These don't seem like warnings - more like audit entries.  Is this the right classification?

Reply:
whoops, not sure happened there, but I recall reapplying this change.. but it wasn't supposed to be here/like this!
Meant to be on the exception on the catch